### PR TITLE
use logging lib instead of print, use logging.exception

### PR DESCRIPTION
**Pull request by xtaje**
A couple things that might be worth pointing out, in the spirit of  being Enterprisey:
* using the logger args instead of f-strings will prevent a buggy `__str__` implementation from crashing your program
* `logging.exception` will include the traceback automatically

### CODE REVIEW

File: chapter_09_commands.asciidoc [hunk 1]
```diff
@@ -426,7 +426,7 @@ def handle_event(event: events.Event, uow: unit_of_work.AbstractUnitOfWork):
             ):
 
                 with attempt:
-                  print('handling event', event, 'with handler', handler)
+                  logging.debug('handling event %s with handler %s', event, handler)
                   handler(event, uow=uow)
         except RetryError:
             print('Failed to handle event, retrying')
```
Comment (xtaje) [inline chapter_09_commands.asciidoc]: This catch black seems wrong, but I've put in another PR to fix.

**Comment by hjwp** [comment]
great.  will just need to do a bit of messing about with git branches and tests to get it to merge.  thanks for the associated prs on the code repo, they help.
